### DISCUSSION TITLE
Plugins not loaded on Lazy Vim

### DISCUSSION THREAD

**Discussion by DoAbner** [LazyVim/LazyVim#325] [Q&A] [answered]
Hello,

I am a newbie to Lazy Vim, I have tried to reinstall many times, but my plugin cannot load, even though it was installed before.
This situation persisted for several times when I tried to install them, I also couldn't find a helpful answer on the Internet.

I installed it on windows 10: NeoVim version 0.8.3

Thank you, gentlemen.

![image](https://user-images.githubusercontent.com/97525283/220623119-b372bde4-e170-41f8-8ccd-04f6f98ec821.png)

**Comment by yusufaktepe** [comment, marked as answer]
Which plugin?
If you're referring to the plugins under the "Not Loaded" list, those will load automatically when the conditions (`event`, `keys`...) are met. See: [lazy.nvim#-plugin-spec](https://github.com/folke/lazy.nvim#-plugin-spec)

**Reply by DoAbner** [reply]
No sir, everything is already installed, I also looked for them and they are all available at nvim-data. looks like it had some problem loading the plugin.

**Reply by dpetka2001** [reply]
As the previous person said those plugins are installed but are only loaded under some conditions(some key press, some event, autocmd, some lsp server or auto completion etc). It's perfectly normal.

**Reply by thoroc** [reply]
Well that's a tad confusing
